pkg/token/README.MD

```diff
@@ -0,0 +1,38 @@
+Instruction for recording response using [GO-VCR](https://github.com/dnaeon/go-vcr) for unit test by jonachang
+# Things to know if you want to record new recording
+* All the recorded responses have been saved under folder```/testdata```
```

Comment:
nit: single backtick ` will do.

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

Reply:
Fixed